modifier/perimeter.go

```diff
@@ -54,38 +54,40 @@ func Perimeters(layer data.PartitionedLayer) ([][][]data.LayerPart, error) {
 	return nil, nil
 }
 
-func (m perimeterModifier) Init(model data.OptimizedModel) {}
+func (m perimeterModifier) Init(_ data.OptimizedModel) {}
 
-func (m perimeterModifier) Modify(layerNr int, layers []data.PartitionedLayer) error {
-	// Generate the perimeters.
-	c := clip.NewClipper()
-	insetParts := c.InsetLayer(layers[layerNr].LayerParts(), m.options.Printer.ExtrusionWidth, m.options.Print.InsetCount)
+func (m perimeterModifier) Modify(layers []data.PartitionedLayer) error {
+	for layerNr := range layers {
```

Comment:
Personally, I'd write `for layerNr, _ := range layers` here to avoid confusion, but that isn't necessary.

Reply:
don't think that's needed if the variable name states clearly what it is